### ISSUE TITLE
Create an intro document for new volunteers

### ISSUE THREAD

**Issue by atalyaalon** [hasadna/anyway-data-science#24]
1. Add intro information about the project
2. Add useful links and info for all (redash, data directory, etc)
3. Add useful links and info for the data team (github directory, notebooks, etc)

**Comment by eliadl** [comment]
@elashahar01 and I discussed how to do this.
We decided to create two docs:
1. General orientation about HaSadna and the Anyway project
2. Technical intro for Anyway Data Scientists

The two docs will reference each other at the very top so each doc can keep serving its single purpose, and provide more focus for the reader.

I've worked on a draft for the latter (technical doc) here, feel free to fork and\or suggest edits:
https://docs.google.com/document/d/1Z2BeWMLi3O79CP7M0vIVtFY0k1DAk_tBxnZpXv8K6a0/edit?usp=sharing
